### ISSUE TITLE
CSS should be calendar specific

### ISSUE THREAD

**Issue by neillshurville** [zackify/react-calendar#9]
I'm not sure why your css rules are generic?
They should all relate to a calendar ID so they only effect the calendar component
As it stands, anyone using your project will have the css rules applied to their entire project.

E.g. this..

```
h1,
h2,
h3,
h4,
h5,
h6 {
  font-size: inherit;
  font-weight: inherit;
}
```

should be something like...

```
#react-calendar h1, #react-calendar h2, #react-calendar h3, #react-calendar h4, #react-calendar h5, #react-calendar h6 {
  font-size: inherit;
  font-weight: inherit;
}
```
Or else you might effect the styling of other, unrelated pages in my app...

**Comment by zackify** [comment]
Sorry about this. I’m using tailwind and forgot it included some generic class definitions. 

Tailwind supports adding a prefix to all of its classes, I will do that and make a new version to solve the problem

**Comment by neillshurville** [comment]
No problem at all, thanks for the reply. We love this project except for the above issue!

**Comment by zackify** [comment]
Thanks. I’ll try to get to this on Friday.

**Comment by matteoMiglio** [comment]
Hi, any update on this issue? Thank you @zackify

**Comment by zackify** [comment]
Sorry been busier than expected. Hopefully I can get to it next week.

**Comment by smai-f** [comment]
I'm happy to contribute this if you'd like the help @zackify ?

**Comment by shermannws** [comment]
@zackify Thank you! Am looking forward to this update too. Cheers!

**Comment by shermannws** [comment]
Hi, I implemented a calendar organiser app for a school project recently. After I import the CSS, it is messing up with a lot of my other CSS styles in the application. Am wondering when this could be fixed! Thank you in advance! :D

**Comment by zackify** [comment]
With the holiday I’ve been busy, but I took time off work later this week and I hope to address the open issues with the project before next week. Fingers crossed

**Comment by shermannws** [comment]
Happy holidays to you buddy! Have a good break too before summer ends. Appreciate your help and looking forward :D

**Comment by zackify** [comment]
```
npm install @zach.codes/react-calendar@0.3.0-beta
```

I released a beta version that fixes this issue. Please try it out.

You will want to import the css from 

```
import '@zach.codes/react-calendar/dist/calendar-tailwind-no-reset.css';
```
to skip adding the reset code. It will be up to you to compare it with the main calendar-tailwind.css file based on the reset code already in your app

**Comment by shermannws** [comment]
Thank you! It is working for me! Cheers

**Comment by zackify** [comment]
Cool, I published 0.3.0, it's the same code as the beta.
